core/dbt/graph/selector.py

```diff
@@ -88,8 +96,8 @@ def split_specs(node_specs: Iterable[str]):
     return specs
 
 
-def get_package_names(graph):
-    return set([node.split(".")[1] for node in graph.nodes()])
+def get_package_names(nodes):
+    return set([node.split(".")[1] for node in nodes])
```

Comment:
👍  this method shouldn't have existed at all since nothing but a test uses it, but this is a good re-purposing!

Reply:
I know but I couldn't live with these dup here 😆 : https://github.com/fishtown-analytics/dbt/blob/dfb72ccb2eddd3ea33cb1456d2c7bb310183c202/core/dbt/graph/selector.py#L193